content/docs/user-guide/experiment-management/running-experiments.md

```diff
@@ -125,10 +125,19 @@ $ dvc exp run -S learning_rate=0.001 -S units=128  # set multiple params
 [parameters files]:
   /doc/user-guide/project-structure/dvcyaml-files#parameters-files
 
+<admon icon="book">
+
+See [Hydra composition](/doc/user-guide/experiment-management/hydra-composition)
+for more advanced configuration options via parameter overrides (change, append,
+or remove, or use "choice" sets and ranges).
```

Comment:
This might be a little pedantic 👼 but I worry that this conflates different Hydra features. We are trying to separate different Hydra features in the docs (see https://iterative.ai/blog/dvc-hydra-integration#what-dvc-users-gain-from-the-integration for more explanation):
1. Overrides (change, append, or remove)
2. Use "choice" sets and ranges for grid search
3. Composition

The first two are used implicitly by `dvc exp run` regardless of whether `hydra.enabled = true`.

Reply:
> The first two are used implicitly by dvc exp run regardless of whether hydra.enabled = true.

I didn't know that. We should def. clarify somewhere but not sure where. Maybe better to follow-up with an issue?